### ISSUE TITLE
We should add which features are needed for a MVP

### ISSUE THREAD

**Issue by JuanGaray93** [ariedro/ey-es-el-pluggo#1]
Here's a small list off the top of my head:

- Users, which include
   - Usernames
   - Avatars (unlockable by level with points)
   - XP (which determines the level)
   - Points (given after each level up)
- An XP system. We should define what actions give a user XP, if there's a cap per day, etc.
- Rooms.
    - Each room should have different roles for its users: creator, admin, mod, dj, banned, etc.
    - Roles should determine possible user actions.
    - This includes video playback, sync and user queuing
- Playlists for each user, ability to import from youtube and perhaps other services like Spotify (also import playlists pls)
- A live multichat for a server's users

**Comment by JuanGaray93** [comment]
Perhaps we could tree these requirements and start working on a single room first, then generic users and a chatroom, then playback, usernames and roles, etc

**Comment by ariedro** [comment]
I agree, how about you add the same list that you just made and add it to the `README.md`, and we would add items to it in the future?

Also be welcome to add yourself in the `Contributors` section, with the roles you'll be doing in the project.

> Perhaps we could tree these requirements and start working on a single room first, then generic users and a chatroom, then playback, usernames and roles, etc

Yes, I was going to make a very simple working app with the basic functionalities the other day from we should work upon on but I got straw, I'll be pushing that in the following days.
